### PR TITLE
Northwest class 4-Humail Khan - JS-Core 1-Week 2 Coursework

### PR DESCRIPTION
**Pull request by humailhasankhan**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

# Notes

- What did you find easy?

- What did you find hard?

- What do you still not understand?

- Any other notes?

### CODE REVIEW

File: exercises/D-logical-operators/exercise2.js [hunk 1]
```diff
@@ -5,7 +5,34 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(number) {
+  if (number < 0) {
+    return true;
+  }
+  return false;
+}
```
Comment (Ash-97) [inline exercises/D-logical-operators/exercise2.js:13]: Hi Humail, I really like your use of IF statement in this whole exercise including the other functions too whereas you could have just done a 'return' with the condition but writing an IF statement makes it pretty clear for other users in terms of readability.

File: exercises/E-conditionals/exercise.js [hunk 2]
```diff
@@ -6,9 +6,14 @@
...
-var name = "Daniel";
-var danielsRole = "mentor";
+const name = "Daniel";
+const danielsRole = "mentor";
 
+if (danielsRole === "mentor") {
+  console.log("Hi, I'm Daniel, I'm a mentor.");
+} else {
+  console.log("Hi, I'm Daniel, I'm a student.");
+}
```
Comment (Ash-97) [inline exercises/E-conditionals/exercise.js:16]: Nice to see that both of us came up with a different way of doing this simple exercise. Here is how I did it just to share different solutions:

const name = "Daniel";
const danielsRole = "mentor";

function classRole(danielsRole) {
  if (danielsRole === "mentor") {
    return "I'm a mentor.";
  } else return "I'm a student.";
}

console.log("Hi, I'm " + name + ", " + classRole(danielsRole));

**Comment by Ash-97** [comment]
Looked through most of your homework and it looks really good. All of your solutions are very clear and concise so well done.

File: exercises/D-logical-operators/exercise2.js (hunk 1 above)
Comment (Maziarmajd) [reply exercises/D-logical-operators/exercise2.js:13]: nice job Humail. your code is clean and organized and I always enjoy your work

File: exercises/E-conditionals/exercise.js (hunk 2 above)
Comment (Maziarmajd) [reply exercises/E-conditionals/exercise.js:16]: I agree with Ash-97. I did this like Ash and it is great seeing you did it in different way.

File: exercises/D-logical-operators/exercise2.js (hunk 1 above)
Comment (humailhasankhan) [reply exercises/D-logical-operators/exercise2.js:13]: Thank you Azan and Maziar. Really appreciate your comments. Thank you for the reviews.

> nice job Humail. your code is clean and organized and I always enjoy your work



> Hi Humail, I really like your use of IF statement in this whole exercise including the other functions too whereas you could have just done a 'return' with the condition but writing an IF statement makes it pretty clear for other users in terms of readability.

Comment (ahmd202) [reply exercises/D-logical-operators/exercise2.js:13]: Great work Humail as usual .

**Pull request closed without merging**